ibm/mas_devops/roles/mariadb/README.md

```diff
@@ -0,0 +1,68 @@
+# Mariadb
+=====
+
+This role provides support to install and configure MariaDB 
+
+* Install MariaDB
+
+Role Variables
+--------------
+
+### MAS core instance
+Action to be performed by mariadb role. Valid values are `string`.
```

Comment:
The descriptions in all of these parameters are not correct and should be updated